Adjust alignment of icons in 2005 interface style and fix inventory icon

sadegr opened the pull request:
I noticed that the alignment of a few of the icons in the 2005 interface style were off by a few pixels (most notably the music icon being too far down and the log out icon being too far to the right). I've gone ahead and updated the alignments of all of them to match the following screenshot (supposedly taken in 2006, but the HUD did not change between 2005/2006 AFAIK):
![TrYePoN 1](https://user-images.githubusercontent.com/20391979/59546016-6f7b6400-8ef5-11e9-9ddc-b99b9f2ea219.png)
[Source](https://www.reddit.com/r/2007scape/comments/2me0bd/can_we_revisit_the_2006_client_for_the_2nd/)

Also, the inventory icon currently used in the 2005 interface is much smaller than it was originally in 2005, so I've updated the sprite. The new sprite was ripped from the above screenshot as well (both the screenshot and sprite are PNGs). I hope there aren't any copyright issues in doing so, but since the other 2005 interface icons were presumably also either ripped from old screenshots or obtained from some client cache, I figured this would be ok.

Before:
![before](https://user-images.githubusercontent.com/20391979/59546090-6c34a800-8ef6-11e9-916d-7158de6888cf.png)

After:
![after](https://user-images.githubusercontent.com/20391979/59546121-bddd3280-8ef6-11e9-97d1-fc73f2c52ccc.png)

Adam- commented:
I have noticed this too, particularly the inventory icon. I'm not sure where we got our existing icon but I think it was from the existing cache? Maybe someone can dump it from a 317 cache and we can compare what we have now with that. CC @Joshua-F

Joshua-F commented:
Here's all of the images directly exported from a (close to) 317 cache.

![sprite_21_12](https://user-images.githubusercontent.com/924482/59547679-516b2f00-8f08-11e9-94e9-6defd3980a3d.png)
![sprite_21_0](https://user-images.githubusercontent.com/924482/59547680-516b2f00-8f08-11e9-86c9-519079fcfbb9.png)
![sprite_21_1](https://user-images.githubusercontent.com/924482/59547681-516b2f00-8f08-11e9-9fcd-571af9ad88e8.png)
![sprite_21_2](https://user-images.githubusercontent.com/924482/59547682-5203c580-8f08-11e9-9f7a-9b11c527b68a.png)
![sprite_21_3](https://user-images.githubusercontent.com/924482/59547683-5203c580-8f08-11e9-99fd-f5a6400fed29.png)
![sprite_21_4](https://user-images.githubusercontent.com/924482/59547684-5203c580-8f08-11e9-93c0-cc2e6ae45035.png)
![sprite_21_5](https://user-images.githubusercontent.com/924482/59547685-5203c580-8f08-11e9-9d43-dc488a6ebd5b.png)
![sprite_21_6](https://user-images.githubusercontent.com/924482/59547686-5203c580-8f08-11e9-8493-4848443f8d17.png)
![sprite_21_7](https://user-images.githubusercontent.com/924482/59547687-5203c580-8f08-11e9-8d74-4c869df13dc0.png)
![sprite_21_8](https://user-images.githubusercontent.com/924482/59547688-529c5c00-8f08-11e9-8196-d420e105424d.png)
![sprite_21_9](https://user-images.githubusercontent.com/924482/59547689-529c5c00-8f08-11e9-8572-8d52cdae6e02.png)
![sprite_21_10](https://user-images.githubusercontent.com/924482/59547690-529c5c00-8f08-11e9-8e74-8edae65d77a2.png)
![sprite_21_11](https://user-images.githubusercontent.com/924482/59547691-529c5c00-8f08-11e9-854a-858d19255d45.png)

[sideicons.zip](https://github.com/runelite/runelite/files/3292769/sideicons.zip)

sadegr commented:
Thanks @Joshua-F . I've used the inventory icon from that zip file, although I had to make a few adjustments to it:
1. It had a much higher DPI than the other HUD icons in the RuneLite repo, so I've adjusted the DPI to match the others
2. It had a different resolution than the other icons in the RuneLite repo, causing the client to display it scaled up. I've changed the resolution to match the others

Screenshot:
![2019-06-15_20-32-37](https://user-images.githubusercontent.com/20391979/59557646-e0318780-8fac-11e9-9ac7-6d43824ed48b.png)
